src/openforms/contrib/kvk/validators.py

```diff
@@ -84,38 +83,43 @@ def __call__(self, value: str) -> bool:
         return True
 
 
-@register("kvk-kvkNumber", verbose_name=_("KvK number"), for_components=("textfield",))
+@register("kvk-kvkNumber")
 @deconstructible
-class KVKNumberRemoteValidator(KVKRemoteBaseValidator):
+class KVKNumberRemoteValidator(BasePlugin[str], KVKRemoteValidatorMixin):
     query_param = "kvkNummer"
     value_label = _("KvK number")
 
-    def __call__(self, value, submission):
+    verbose_name = _("KvK number")
+    for_components = ("textfield",)
+
+    def __call__(self, value: str, submission):
         validate_kvk(value)
-        super().__call__(value)
+        super().validate(value)
```

Comment:
Because of MRO reasons, mixins go to the left, base classes to the right.

It's also weird to use `super()` while you know that you're putting a mixin in there that provides a `validate` method.

```py
@register("kvk-kvkNumber")
@deconstructible
class KVKNumberRemoteValidator(KVKRemoteValidatorMixin, BasePlugin[str]):
    query_param = "kvkNummer"
    value_label = _("KvK number")

    verbose_name = _("KvK number")
    for_components = ("textfield",)

    def __call__(self, value: str, submission):
        validate_kvk(value)
        self.validate(value)
```

should be the correct implementation.